sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/CHANGELOG.md

```diff
@@ -1,7 +1,13 @@
 # Release History
 
-## 5.0.0-beta.4 (Unreleased)
+## 5.0.0-beta.4 (2021-06-08)
 
+### Added
```

Comment:
not sure if it matters to you, but the ordering of `Added` and `Key Bug Fixes` is the opposite here than in the Event Grid readme.

Reply:
Is it? The latest EventGrid release only has an Added section.